source/faucet/main.d

```diff
@@ -347,7 +399,7 @@ public class Faucet : FaucetAPI
 
     public void setup (uint count)
     {
-        while (!this.state.update(this.client, Height(0)))
+        while (!this.state.update(this.clients[0], Height(0)))
```

Comment:
Would have been better to separate support for multiple clients in its own commit.

Reply:
Split the multi-client support into separate commit.